common/static/coffee/spec/discussion/view/discussion_thread_view_spec.coffee

```diff
@@ -102,40 +157,66 @@ describe "DiscussionThreadView", ->
                 course_settings: DiscussionSpecHelper.makeCourseSettings()
             )
 
+        describe "responses", ->
+            it "can post a first response", ->
+                # Initially render a test post (made by someone else) with zero responses
+                renderWithTestResponses(@view, 0)
+                postResponse(@view, 1)
+                expect(@view.$(".forum-response").length).toBe(1)
+                # At this point, there are 2 DiscussionContentViews, the main post and the response.
+                # Each an .action-edit button, but only 1 (the response) should be available.
+                expect(@view.$(".post-actions-list").find(".action-edit").parent(".is-hidden").length).toBe(1)
+                expect(@view.$(".response-actions-list").find(".action-edit").parent().not(".is-hidden").length).toBe(1)
+
+            it "can post a second response", ->
+                # Initially render a test post (made by someone else) with a single response (made by the current learner)
+                renderWithTestResponses(@view, 1)
+                expect(@view.$(".forum-response").length).toBe(1)
+                # Post should not be editable, response should be
+                expect(@view.$(".post-actions-list").find(".action-edit").parent(".is-hidden").length).toBe(1)
+                expect(@view.$(".response-actions-list").find(".action-edit").parent().not(".is-hidden").length).toBe(1)
+
+                # Now make a second response. Prior to TNL-3788, a bug would hide the edit button for the first response
+                postResponse(@view, 2)
+                expect(@view.$(".forum-response").length).toBe(2)
+                # Post should not be editable, responses should be
+                expect(@view.$(".post-actions-list").find(".action-edit").parent(".is-hidden").length).toBe(1)
+                expect(@view.$(".response-actions-list").find(".action-edit").parent().not(".is-hidden").length).toBe(2)
```

Comment:
@cahrens I've added some comments to these new tests and made the selectors more specific, does this seem clearer?